recipe/meta.yaml

```
@@ -1,55 +1,65 @@
 {% set name = "tabula-py" %}
-{% set version = "2.6.0" %}
+{% set version = "2.10.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b10b83d15e7209fcf65c64dee064bdb2d50b06a7e4e28fc9974b92c9ef14c5cc
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name|replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: 75968a83fe978e5d56ccf23f0f0255a459c256b7b52db7cabe5ac795bb3b12df
 
 build:
   number: 0
-  skip: True  # [py<37]
-  # openjdk >=8 isn't available on ppc64le, s390x
-  skip: True  # [ppc64le or s390x]
+  skip: true  # [py<39]
   script:
-    - python -m pip install --no-deps --ignore-installed .
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
-    - setuptools
-    - setuptools_scm
-    - wheel
+    - setuptools >=45
+    - setuptools-scm >=6.2
```

https://github.com/chezou/tabula-py/blob/v2.10.0/pyproject.toml#L2
```suggestion
    - setuptools-scm >=6.2
    - toml
```